server/controllers/match.rb

```diff
@@ -139,6 +139,7 @@ class MatchController < ApplicationController
     json_halt 404, "Member #{member_id} not found as part of match #{match_id}, group #{group_id}, season #{season_id}" if user_season_match.nil?
     if params.has_key? "status"
       user_season_match.won = params[:status]
+      user_season_match.game_wins = params[:game_wins].to_i
```

Comment:
Realistically this should check if the `game_wins` key is present before calling `.to_i` on it (or risk a 500 due to nil); along with this, I wouldn't cast the string, I would let the model figure out if it's a valid value or not.